website/templates/report.html

```diff
@@ -294,6 +295,19 @@ <h3 class="text-xl font-bold leading-none text-red-500">Latest Issues</h3>
                          class="w-[75%] ml-6 md:ml-0 flex items-end mt-6 flex-wrap min-h-[180px]"></div>
                 </div>
             </div>
+            <div>
+                <div class="cve_no">
+                    <label for="cve_no">CVE Number</label>
+                    <input class="mt-4  ml-2 w-100 rounded border-2"
+                           type="text"
+                           placeholder="CVE-XXXX-XXXX..."
+                           id="cve_no_input"
```

Comment:
Another reason to use better naming -- `no input` 🤷‍♂️

Reply:
ahhh yess got it changes it to `cve_id_input`